cms/djangoapps/contentstore/views/transcripts_ajax.py

```diff
@@ -165,10 +166,8 @@ def download_transcripts(request):
         log.debug('transcripts are supported only for video" modules.')
         raise Http404
 
-    filename = 'subs_{0}.srt.sjson'.format(subs_id)
-    content_location = StaticContent.compute_location(item.location.course_key, filename)
     try:
-        sjson_transcripts = contentstore().find(content_location)
+        sjson_transcripts = Transcript.asset(item.location, subs_id)
```

Comment:
Not really a question for this PR, but in general, it's better to prefer a module for namespacing, rather than a class with all `staticmethods`.